### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by piperun** [MerlijnWajer/pyroTorrent#4]
Given the lack of commits, I would assume this project is at the very least on hold?

**Comment by MerlijnWajer** [comment]
I have not used pyroTorrent in a while, I switched to transmission on one machine. On my machine at home I already have rTorrent set up again, but it's not running until I finished my grsecurity RBAC for it. Once that is done, I'll make sure to set up pyroTorrent again.

Have you tried to see if pyroTorrent still works? I don't think the XMLRPC changes that often. I'm happy to merge fixes/features as well as take feature requests.

**Comment by piperun** [comment]
Oh that's good to hear, at the moment I'm using the most common setup of rtorrent + rutorrent, but I could try and setup a testbox with pyrotorrent, albeit later.

**Comment by MerlijnWajer** [comment]
Did you set this up? I know that at least all the rtorrent API stuff still works fine, because somebody I know is using pyroTorrent for his (large scale) torrent project.

**Comment by MerlijnWajer** [comment]
Do you have any specific wishes? Did you try the project in the meantime?

I guess one of the first things to do is to make it (even) easier to run:

* Upgrade to newer flask
* Add some setup.py file and package it on pip
* Update instructions

And then figure out what features we may want to add

**Comment by piperun** [comment]
Oh apologize for late response, unfortunately I currently use the classical rutorrent + rtorrent combo, but as the torrent list grows larger (and being in desperate need of a more powerful torrent client/addon) I would say that in the future I would like to try it out.
